vmdb/app/models/host.rb

```diff
@@ -1416,7 +1416,11 @@ def connect_ssh(options={})
     options[:authentication_prompt_delay] = prompt_delay unless prompt_delay.nil?
 
     users = su_user.nil? ? rl_user : "#{rl_user}/#{su_user}"
-    $log.info "host.connect_ssh: Initiating SSH connection to Host:[#{self.name}] using [#{hostname}] for user:[#{users}].  Options:[#{options.inspect}]"
+    # Obfuscate private keys in the log with ****, so it's visible that field was used, but no user secret is exposed
+    logged_options = options.dup
+    logged_options[:key_data] = "****" if logged_options[:key_data]
```

Comment:
Prefer 8 stars which is the standard via of an obfuscated password  (Other option is the phrase "[FILTERED]" if it's in the log.)